R/interface.R

```diff
@@ -692,7 +692,7 @@ incident_edges <- function(graph, v, mode = c("out", "in", "all", "total")) {
 
   on.exit(.Call(R_igraph_finalizer))
 
-  res <- .Call(R_igraph_incident_edges, graph, vv, mode)
+  res <- incident_impl(graph, vid = vv, mode = mode)
```

Comment:
Above, remove the `- 1` in the calculation of `vv`, and below remove the correction of +1 for indices.